<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="channelThumbnail">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>231</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="channelTitle">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Channel Title</string>
   </property>
  </widget>
  <widget class="QLabel" name="numberVidLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>411</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Num of Vids</string>
   </property>
  </widget>
  <widget class="QLabel" name="durationLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>421</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Duration</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
